### PR TITLE
Add spanish translation to Clicker Game workshop

### PR DESCRIPTION
**Pull request by davidballezaa**
- Added es-xl file with spanish translation
- Added `locales: 'es-xl'` line to README's metadata
- There were 2 new lines at end of README, deleted one of them

### CODE REVIEW

**Comment by davidballezaa** [comment]
Could you please review the translation @J-cordz?

File: workshops/clicker_game/es-xl.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+```jsx
+<h1>{count}</h1>
+```
+
+Ahora todo debería de funcionar... ¿o no? Intenta y mira lo que pasa.
+
+[Un video mostrando números siendo impresos en la consola del navegador cada que damos clic, sin que se actualice la página](https://cloud-rbqvdgtkm.vercel.app/0screen_recording_2020-11-20_at_10.04.52_pm.gif)
+
+Como puedes ver, la variable está incrementando cada que damos clic (podemos saberlo porque vemos la cuenta actualizada siendo impresa en la consola). Pero, por alguna razón, nuestra página no se actualiza. ¿Porqué? Bueno, tenemos que decirle explícitamente a React que re-renderice un cierto componente cuando queramos actualizarlo - si no, se quedará igual que como estaba en el renderizado inicial. ¿Cómo hacemos eso? Ahí es donde el hook `useState()` de React entra al juego.
```
Comment (J-cordz) [inline workshops/clicker_game/es-xl.md:72]:
```suggestion
Como puedes ver, la variable está incrementando cada que damos clic (podemos saberlo porque vemos la cuenta actualizada siendo impresa en la consola). Pero, por alguna razón, nuestra página no se actualiza. ¿Por qué? Bueno, tenemos que decirle explícitamente a React que re-renderice un cierto componente cuando queramos actualizarlo - si no, se quedará igual que como estaba en el renderizado inicial. ¿Cómo hacemos eso? Ahí es donde el hook `useState()` de React entra al juego. 

```

Comment (J-cordz) [reply workshops/clicker_game/es-xl.md:72]: For `¿Porqué?` to be in the interrogative mood, it has to be separated like this: `¿Por qué?`

**Review by J-cordz — CHANGES_REQUESTED** [review]
There's only one thing to change, besides that everything looks perfect!

**Comment by J-cordz** [comment]
Hello @davidballezaa thanks for the help translating! I marked a suggestion for a word so that it is translated correctly as a question. Apart from that, everything is great!

**Comment by davidballezaa** [comment]
Good catch Javi! thank you so much

**Review by sampoder — APPROVED** [review]
🙌 🚀

**Pull request merged**